tests/acceptance/features/apiAntivirus/antivirus.feature

```diff
@@ -127,3 +127,73 @@ Feature: antivirus
+    When user "Brian" uploads a file inside space "Shares" with content "X5O!P%@AP[4\PZX54(P^)7CC)7}$EICAR-STANDARD-ANTIVIRUS-TEST-FILE!$H+H*" to "/uploadFolder/virusFile.txt" using the WebDAV API
+    Then the HTTP status code should be "201"
+    And user "Brian" should get a notification with subject "Virus found" and message:
+      | message                                                                   |
+      | Virus found in virusFile.txt. Upload not possible. Virus: Win.Test.EICAR_HDB-1 |
+    And as "Brian" file "/uploadFolder/virusFile.txt" should not exist
+
+
+  Scenario Outline: upload a file with virus to a group share
+    Given using <dav-path-version> DAV path
+    And user "Brian" has been created with default attributes and without skeleton files
+    And group "group1" has been created
+    And user "Brian" has been added to group "group1"
+    And user "Alice" has created folder "uploadFolder"
+    And user "Alice" has shared folder "uploadFolder" with group "group1"
+    And user "Brian" has accepted share "/uploadFolder" offered by user "Alice"
+    When user "Brian" uploads file "filesForUpload/filesWithVirus/eicar.com" to "/Shares/uploadFolder/virusFile.txt" using the WebDAV API
+    Then the HTTP status code should be "201"
+    And user "Brian" should get a notification with subject "Virus found" and message:
+      | message                                                                        |
+      | Virus found in virusFile.txt. Upload not possible. Virus: Win.Test.EICAR_HDB-1 |
+    And as "Brian" file "/uploadFolder/virusFile.txt" should not exist
+    And as "Alice" file "/uploadFolder/virusFile.txt" should not exist
+    Examples:
+      | dav-path-version |
+      | old              |
+      | new              |
+
+
+  Scenario: upload a file with virus to a group share using spaces dav endpoint
+    Given using spaces DAV path
+    And user "Brian" has been created with default attributes and without skeleton files
+    And group "group1" has been created
+    And user "Brian" has been added to group "group1"
+    And user "Alice" has created folder "uploadFolder"
+    And user "Alice" has shared folder "uploadFolder" with group "group1"
+    And user "Brian" has accepted share "/uploadFolder" offered by user "Alice"
+    When user "Brian" uploads a file inside space "Shares" with content "X5O!P%@AP[4\PZX54(P^)7CC)7}$EICAR-STANDARD-ANTIVIRUS-TEST-FILE!$H+H*" to "/uploadFolder/virusFile.txt" using the WebDAV API
+    Then the HTTP status code should be "201"
+    And user "Brian" should get a notification with subject "Virus found" and message:
+      | message                                                                        |
+      | Virus found in virusFile.txt. Upload not possible. Virus: Win.Test.EICAR_HDB-1 |
+    And as "Brian" file "/uploadFolder/virusFile.txt" should not exist
+    And as "Alice" file "/uploadFolder/virusFile.txt" should not exist
```

Comment:
Note that the step `And as "Brian" file "/uploadFolder/virusFile.txt" should not exist` will alweays pass in these scenarios because the path `/uploadFolder/virusFile.txt` doesn't really exist for `brian` in files root level
```suggestion
    And as "Brian" file "/Shares/uploadFolder/virusFile.txt" should not exist
    And as "Alice" file "/uploadFolder/virusFile.txt" should not exist
```

Reply:
ok